src/main/generated/io/vertx/redis/client/RedisConnectOptionsConverter.java

```diff
@@ -40,16 +40,6 @@ static void fromJson(Iterable<java.util.Map.Entry<String, Object>> json, RedisCo
             obj.setPassword((String)member.getValue());
           }
           break;
-        case "endpoints":
```

Comment:
@Ladicek this still gets removed, is that a breaking change ?

Reply:
It is not removed, it is _moved_, see below. This is to ensure that `endpoints` is serialized last and hence also deserialized last. This prevents information loss.

Previously, `endpoints` was serialized before `endpoint`, but `endpoint` is just the first element of the `endpoints` list. Since deserialization order follows serialization order, `endpoints` would be deserialized first and then `endpoint` would be deserialized, which leaves only a single element in the `endpoints` list. If there were multiple endpoints, only the first of them would be kept; the rest would be lost.

Reply:
oh right, my bad :-)